src/repositories/UserRepository.ts

```diff
@@ -129,4 +147,14 @@ export class UserRepository extends AbstractRepository<UserModel> {
   public async deleteUser(user: UserModel): Promise<UserModel> {
     return await this.repository.remove(user);
   }
+
+  public async saveToken(user: UserModel, token: string): Promise<UserModel> {
+    if (!user.deviceTokens) {
+      user.deviceTokens = []
+    }
+    else {
```

Comment:
shouldn't be else

Reply:
oh wait this isn't here anymore, i deleted this function because it was the exact same as addDeviceToken